nautobot/dcim/filters/__init__.py

```diff
@@ -1767,3 +1771,25 @@ class DeviceRedundancyGroupFilterSet(NautobotFilterSet, StatusModelFilterSetMixi
     class Meta:
         model = DeviceRedundancyGroup
         fields = ["id", "name", "slug", "failover_strategy"]
+
+
+class InterfaceRedundancyGroupFilterSet(BaseFilterSet, NameSlugSearchFilterSet):
+    """Filter for InterfaceRedundancyGroup."""
+
+    class Meta:
+        """Meta attributes for filter."""
+
+        model = InterfaceRedundancyGroup
+
+        fields = ["id", "name", "slug", "description", "virtual_ip"]
+
+
+class InterfaceRedundancyGroupAssociationFilterSet(BaseFilterSet, NameSlugSearchFilterSet):
+    """Filter for InterfaceRedundancyGroupAssociation."""
+
+    class Meta:
+        """Meta attributes for filter."""
+
+        model = InterfaceRedundancyGroupAssociation
+
+        fields = ["id", "group", "interface", "priority"]
```

Comment:
Is it possible to use `fields = "__all__"` on these new filtersets in keeping with our best practices?

Reply:
fixed this

Reply:
Using `fields = "__all__"` is causing a traceback when running `makemigrations`.  I spoke with @bryanculver about this and he suggested that this is fixed in 2.0 and that we should leave this as-is until 2.0.